src/sagemaker/jumpstart/artifacts/script_uris.py

```diff
@@ -119,6 +120,8 @@ def _model_supports_inference_script_uri(
             retrieve the support status for script uri with inference.
         model_version (str): Version of the JumpStart model for which to retrieve the
             support status for script uri with inference.
+        hub_arn (str): The arn of the SageMaker Hub for which to retrieve
```

Comment:
`The arn of the SageMaker Hub from which to retrieve model details.` -- i think this is what you want

can be for another PR